src/videoserver/apps/projects/routes.py

```diff
@@ -370,24 +370,24 @@ def schema_edit(self):
                 'empty': True,
                 'schema': {
                     'width': {
-                        'type': 'integer',
+                        'type': 'float',
                         'min': app.config.get('MIN_VIDEO_WIDTH'),
```

Comment:
The real values when client send to server are `float` number.

Reply:
Is it possible to simply convert float to int on client side before sending a request to the server? 
Imo on server side all crop params must be `int` not `float`.  
P.S. Keep doc strings synchronised with a code.

Reply:
Maybe It make the cropped video isnot exactly. But the wrong is very small. We think it won't effect to user.
Ok, we will keep `integer` type.